data.md

```diff
@@ -337,23 +347,84 @@ One needs to unname the `ValTypes` first before calling the `#take` or `#drop` f
 Strings
 -------
 
+Wasm use a different character escape rule with K, so we need to define the `unescape` function ourselves.
+
+```k
+    syntax String ::= unescape(String)              [function]
+                    | unescape(String, Int, String) [function, klabel(unescapeAux)]
+ // -------------------------------------------------------------------------------
+    rule unescape(S) => unescape(S, 1, "")
```

Comment:
Can we inline some documentation between the rules? So an explanation here that unescaped characters just gets added to the output ...